data_dig.sql

```diff
@@ -0,0 +1,147 @@
+CREATE TABLE mock_user_data (
+			 id INTEGER,
+			 first_name TEXT,
+			 gender TEXT,
+			 hometown TEXT,
+			 birth_date TEXT,
+			 lucky_number INTEGER,
+			 cash INTEGER
+);
+INSERT INTO mock_user_data (id, first_name, gender, hometown, birth_date, lucky_number, cash) VALUES (1, 'Jacquette', 'Female', 'Erong', '4/29/1996', 30, 177);
```

Comment:
A good indentation would be to put each SQL keyword on a new line. So INSERT INTO and VALUES are under each other. It makes it a little more readable.